plugin/src/main/java/org/opensearch/ml/action/register/TransportRegisterModelAction.java

```diff
@@ -171,7 +172,8 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<MLRegi
                 "To upload custom model user needs to enable allow_registering_model_via_url settings. Otherwise please use OpenSearch pre-trained models."
             );
         }
-        registerModelInput.setIsHidden(RestActionUtils.isSuperAdminUser(clusterService, client));
+        boolean isSuperAdmin = isSuperAdminUserWrapper(clusterService, client);
```

Comment:
This method for a single line adds complexity, additional method maintenance and reduced readability. this isn't typically considered best practice unless there's a specific testing requirement, but this static method is not covered in the UT at all.

Reply:
If you notice, I have mocked the behavior in both of the tests:

```
        // Mock super admin check
        doReturn(false).when(transportRegisterModelAction).isSuperAdminUserWrapper(any(), any());
```